agwa-section1-accounts-domains-directory.md

```
@@ -0,0 +1,1324 @@
+
+#### 動的グループの作成手順
+
+```
+Admin コンソール → ディレクトリ → グループ → グループを作成 
+→ 「メンバーシップを動的にする」を選択
+→ クエリを入力（例：user.department == "Sales"）
+→ グループを保存
+```
+
+> ⚠️ **注意**: 動的グループは **Enterprise エディション** または **Cloud Identity Premium** が必要です。
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)